smart-campus-service-impl/src/main/java/hu/unideb/smartcampus/service/ejabberd/SharedRosterServiceImpl.java

```diff
@@ -0,0 +1,132 @@
+package hu.unideb.smartcampus.service.ejabberd;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import hu.unideb.smartcampus.service.ejabberd.request.AddUserRequest;
+import hu.unideb.smartcampus.service.ejabberd.request.CreateGroupRequest;
+import hu.unideb.smartcampus.service.ejabberd.request.DeleteUseRequest;
+import hu.unideb.smartcampus.service.ejabberd.request.GroupRequest;
+import hu.unideb.smartcampus.service.ejabberd.request.InformationRequest;
+
+
+/**
+ * Shared Roster Service Implementation which implements {@link SharedRosterService}.
+ *
+ * <p>
+ * It handles shared roster groups (srg) via REST API which is provided by <b>Ejabberd</b>.
+ * </p>
+ */
+@Service
+public class SharedRosterServiceImpl extends EjabberdService implements SharedRosterService {
```

Comment:
This inheritance is not cool.
You are just using from the super class static properties.(Anyway the superclass will be removed).